go/history/history_test.go

```diff
@@ -59,32 +53,20 @@ func TestLatest(t *testing.T) {
 
 	// Add first value.
 	h.Add(mod10(1))
-	if got, want := int(h.Records()[0].(mod10)), 1; got != want {
-		t.Errorf("h.Records()[0] = %v, want %v", got, want)
-	}
-	if got, want := int(h.Latest().(mod10)), 1; got != want {
-		t.Errorf("h.Latest() = %v, want %v", got, want)
-	}
+	assert.EqualValues(t, 1, int(h.Records()[0].(mod10)))
```

Comment:
The point of `EqualValues` is that you don't need casting for compatible values.
```suggestion
	assert.EqualValues(t, 1, h.Records()[0].(mod10))
```